shed/event_streams.py

```diff
@@ -735,7 +737,7 @@ def update(self, x, who=None):
             tup = tuple(buf.popleft() for buf in self.buffers)
             self.condition.notify_all()
             self.prior = tup
-            return self.emit(tup)
+            return self.emit(self.curate_streams(tup))
```

Comment:
agreed. i guess i could just modify this PR a little later as well, unless you'd rather take care of this one. just let me know!